pygmt/src/grdcut.py

```diff
@@ -91,10 +98,21 @@ def grdcut(grid, outgrid: str | None = None, **kwargs):
     >>> # 12° E to 15° E and a latitude range of 21° N to 24° N
     >>> new_grid = pygmt.grdcut(grid=grid, region=[12, 15, 21, 24])
     """
+    inkind = data_kind(grid)
+    outkind = "grid"
+    if inkind == "image":
+        outkind = "image"
```

Comment:
Simplify to one-line
```suggestion
    outkind = "image" if inkind == "image" else "grid"
```